### PR TITLE
Add `to_bytes` with some test and example.

### LINKED ISSUE #34: How can I get byte from qrcode image?
I can see `to_file` but in my case I will need `byte` as a http response `from_byte` for a return but can't find anyway to do it.

### PR DESCRIPTION
**Pull request by katopz**
It's a quick addition for test and examples, feel free to hit me. ;)

Edit: Closes #34

### CODE REVIEW

File: src/convert/image.rs
Comment (erwanvivien) [inline src/convert/image.rs:1]: Could you fixup the previous commit with this one (make it one commit)

**Comment by erwanvivien** [comment]
Thanks for the PR 🙏

**Review by erwanvivien — APPROVED** [review]

**Comment by katopz** [comment]
I've to push force to make it 1 commit sorry! 😅

**Comment by erwanvivien** [comment]
> I've to push force to make it 1 commit sorry! 😅

Perfectly fine :)

**Comment by erwanvivien** [comment]
I'll do some minor changes to my code where I spotted issues and push a v0.10 soon @katopz, if that's ok 👍 
Will ping when avail.

**Comment by erwanvivien** [comment]
@katopz It's up :)

**Comment by katopz** [comment]
🦀🎉

**Pull request merged**